package.json

```diff
@@ -20,6 +20,7 @@
     "@okta/okta-sdk-nodejs": "^6.5.0",
     "chalk-table": "^1.0.2",
     "generate-password": "^1.7.0",
+    "node-fetch": "^3.2.10",
```

Comment:
What was this used for?

Reply:
It was a result of a full module import, see the IE-24 merge request for the fixing of this issue.